src/components/styles/DetailsPage.css

```diff
@@ -0,0 +1,52 @@
+.container {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+  padding: 1rem;
+}
+
+.back-link {
+  position: absolute;
+  top: 2rem;
+  left: 2rem;
+}
+
+.left-arrow-icon {
+  font-size: 2rem;
+}
+
+.card {
+  background-color: #fff;
+  border-radius: 0.5rem;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+  padding: 2rem;
+  max-width: 500px;
+  width: 100%;
+  text-align: center;
+}
+
+.card img {
+  max-width: 100px;
+  margin-bottom: 1rem;
+}
+
+.card p {
+  margin-bottom: 1rem;
```

Comment:
- [ ] Same as here, please make sure you follow the layout, either you can change the color to your preferred one but the page wireframe must be identic to the design

<img width="576" alt="2" src="https://github.com/kessie2862/Metrics-Webapp/assets/77843050/2dc6ea75-d36a-411f-a5fe-5ad13b09fa08">

Reply:
I have made the necessary changes. Thanks 😊